grp/simple.gi

```diff
@@ -887,9 +887,101 @@ local id;
   return DataAboutSimpleGroup(id);
 end);
 
+# Tables for outer automorphisms from Bray/Holt/Roney-Dougal, p. 36/37
```

Comment:
Thanks for including this line.

I have no idea how we deal with referencing inGAP, in general (and whether we have any kind of 'policy' about it): so for anyone who knows about this sort of sutff, should there also be a corresponding reference to this book in the GAP manual/bibliography/website/somewhere else?

Reply:
In practice there is no consistent way this is dealt with, I am afraid. Of course in principle it would be good to at least add a BibTeX entry for this in our master .bib file. Whether and where to cite it is a different question, though...